src/screens/transfer/screen/delegateScreen.js

```diff
@@ -535,7 +535,7 @@ class DelegateScreen extends Component {
     );
 
     const _renderStepTwo = () => (
-      <AnimatedView animation="bounceInRight" delay={500} useNativeDriver>
+      <Animated.View entering={BounceInRight.duration(500)}>
```

Comment:
is adding delay() here required or was a typo before...?

Reply:
yes, I think its a typo. Added delay again